mirrord-cli/src/config.rs

```diff
@@ -144,7 +121,7 @@ pub(super) struct ExecArgs {
     pub no_telemetry: bool,
 
     /// Load config from config file
-    #[clap(short = 'f', conflicts_with_all = &["target", "pod-name"], long, value_parser)]
+    #[clap(short = 'f', conflicts_with_all = &["target"], long, value_parser)]
```

Comment:
I don't think we need `conflicts_with_all` at all, you can load config file and target, it will work just fine from MirrordConfig's perspective